cmd/evmosd/testnet.go

```diff
@@ -95,6 +99,8 @@ func addTestnetFlagsToCmd(cmd *cobra.Command) {
 	cmd.Flags().String(flags.FlagChainID, "", "genesis file chain-id, if left blank will be randomly created")
 	cmd.Flags().String(sdkserver.FlagMinGasPrices, fmt.Sprintf("0.000006%s", cmdcfg.BaseDenom), "Minimum gas prices to accept for transactions; All fees in a tx must meet this minimum (e.g. 0.01photino,0.001stake)")
 	cmd.Flags().String(flags.FlagKeyType, string(hd.EthSecp256k1Type), "Key signing algorithm to generate keys for")
+	cmd.Flags().String(flagBaseFee, "1000000000", "The params base_fee in the feemarket module in geneis")
+	cmd.Flags().String(flagMinGasPrice, "100000000", "The params min_gas_price in the feemarket module in geneis")
```

Comment:
Please update the default values to be the same as current ones
```suggestion
	cmd.Flags().String(flagMinGasPrice, "0", "The params min_gas_price in the feemarket module in geneis")
```

Reply:
done